docs/body.md

```diff
@@ -538,6 +541,19 @@ changes will still be caught without this option enabled, it is
 recommended to leave notify_on_state_change disabled if your
 OpenStack environment is very large.
 
+#### Network and Router Info Events
+
+The OpenStack Infrastructure ZP pulls neutron events for Networks and Routers. However,
+if Neutron is not configured properly to send those events, they cannot be retrieved.
+If you are missing Info events, checks your OpenStack environment's 
+/etc/neutron/neutron.conf. It should have the following:
+
+```
+[oslo_messaging_notifications]
```

Comment:
Wouldn't it make more sense to put the instructions in with the ceilometer_zenoss docs, and just point there?  That way, if we change the plugin, the docs are in the same repo, and we don't also have to update OSI.  Or is some of this relevant even if you're not using that plugin?